mos-platform/atari2600-3e/mapper_3e.c

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 LLVM-MOS Project
+// Licensed under the Apache License, Version 2.0 with LLVM Exceptions.
+// See https://github.com/llvm-mos/llvm-mos-sdk/blob/main/LICENSE for license
+// information.
+
+#include <peekpoke.h>
+#include <mapper.h>
+
+__attribute__((section(".zp.bss"))) bank_index_t __current_bank;
+
+__attribute__((weak)) void set_current_bank(bank_index_t bank_id) {
```

Comment:
Does it actually make sense to have these weak? Can the functionality in this file be meaningfully partially overriden?
I tended to go a bit weak-happy in the early days of the SDK, but library files like this will only be brought in to the link if an undefined symbol reference is fulfilled by it, so all weak buys is allowing you to replace part, but not all, of this library.